lib/ace/ext/papeeria/latex_parsing_context_test.js

```diff
@@ -0,0 +1,28 @@
+if (typeof process !== "undefined") {
+    require("amd-loader");
+    require("../../test/mockdom");
+}
+
+define(function(require, exports, module) {
+    var EditSession = require("ace/edit_session").EditSession;
+    var Editor = require("ace/editor").Editor;
+    var MockRenderer = require("ace/test/mockrenderer").MockRenderer;
+    var LatexParsingContext = require("ace/ext/papeeria/latex_parsing_context");
+    var PapeeriaLatexHighlightRules = require("ace/ext/papeeria/papeeria_latex_highlight_rules").PapeeriaLatexHighlightRules;
+    Tokenizer = require("ace/tokenizer").Tokenizer;
+    var PapeeriaLatexMode = {
+      getTokenizer: function() {
+        return new Tokenizer((new PapeeriaLatexHighlightRules()).getRules());
+      }
+    };
```

Comment:
In fact we normally use 2 spaces for indenting, so I re-indented this file with 2 spaces.